cmd/infra/azure/create.go

```diff
@@ -252,10 +261,23 @@ func createResourceGroup(ctx context.Context, o *CreateInfraOptions, azureCreds
 
 		return *response.ID, *response.Name, existingRGSuccessMsg, nil
 	} else {
```

Comment:
I would expect a user could add tags even if they BYO RG. Would there be a reason not to?

Reply:
No hard feelings on this, but I was more less thinking if you BYO RG then you already had the opportunity to add tags and chose not to.